pages/public_cloud/containers_orchestration/managed_kubernetes/forwarding-audit-logs-to-logs-data-platform/guide.en-gb.md

```diff
@@ -0,0 +1,197 @@
+
+The OVHcloud Managed Kubernetes Service has defined audit policy, enabling you to retrieve logs pertaining to:
+-	requests to authorization resources to help troubleshoot authentication issues
+-	configmap and secret changes in all namespaces at the Metadata audit level
+-	changes to resources at RequestResponse level (maximum verbosity) to create, patch, update and delete verbs
+-	changes to resources at Request level for other verbs
+-	all other requests at the Metadata level
+
+Note:
+-	No logs for requests by the "system:kube-proxy" on endpoints or services
+-	Low level of verbosity (level Metadata) for endpoints containing sensitive data like tokenreview
+-	No logs for any request in the RequestReceived stage
+-	No logs for health checks and requests for apiserver metrics
+
+
+For details about information captured in Kubernetes Audit logs you can refer to the [Kubernetes public documentation](https://kubernetes.io/docs/reference/config-api/apiserver-audit.v1/).
+
+Above, you will find an example of an audit log generated by a Kubernetes cluster. *Note the example is not exhaustive.*
+
+```json
+{
+   "kind":"Event",
+   "apiVersion":"audit.k8s.io/v1",
+   "level":"Metadata",
+   "auditID":"XXXXXXX-XXXX-XXXX-XXXX-XXXXXXXX",
+   "stage":"ResponseStarted",
+   "requestURI":"/api/v1/namespaces/searching-worker/configmaps?......",
+   "verb":"watch",
+   "user":{
+      "username":"system:node:search-workers-node-81097c",
+      "groups":[
+         "system:nodes",
+         "system:authenticated"
+      ]
+   },
+   "sourceIPs":[
+      "10.10.0.43"
+   ],
+   "userAgent":"kubelet/v1.26.4 (linux/amd64) kubernetes/f89670c",
+   "objectRef":{
+      "resource":"configmaps",
+      "namespace":"searching-worker",
+      "name":"kube-root-ca.crt",
+      "apiVersion":"v1"
+   },
+   "responseStatus":{
+      "metadata":{
+
+      },
+      "code":200
+   },
+   "requestReceivedTimestamp":"2024-01-01T11:59:06.428015Z",
+   "stageTimestamp":"2024-01-01T11:59:06.428962Z",
+   "annotations":{
+      "authorization.k8s.io/decision":"allow",
+      "authorization.k8s.io/reason":""
+   }
+}
+```
+
+## How to enable Audit Log Forwarding
+### Using the Manager
+
+#### Create and see existing streams
+
+ You can find your streamId, in Logs Data Platform control panel of your account in Data stream” pan when editing a stream:
+
+ -	Got to the “Data Stream” page of your Logs Data Platform account and “Edit” the target Data Stream
+
+ https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/home
+
+ -	Copy the streamId page of your Logs Data Platform account.
+
+ https://www.ovh.com/manager/#/dedicated/dbaas/logs/{ldp-account-id}/streams/{streamid}/edit
+
+### Using APIs
```

Comment:
Is it possible through Terraform?

Reply:
not yet, I will update the doc when available